test/rules/pure-describe.js

```diff
@@ -0,0 +1,89 @@
+'use strict';
```

Comment:
Why does this has a different filename than the actual rule?

Reply:
Oops, changed the name
- [ ] rename this file